### PR TITLE
[17.0][MIG] sale_order_client_order_ref_mandatory

### PR DESCRIPTION
**Pull request by IriaAlonso**

### CODE REVIEW

**Comment by IriaAlonso** [comment]
@dalonsod

File: sale_order_client_order_ref_mandatory/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,418 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dalonsod) [inline sale_order_client_order_ref_mandatory/static/description/index.html]: Remove this line, as usual

**Review by dalonsod — CHANGES_REQUESTED** [review]
Tested, please take a look at tree views, because the field is duplicated (it seems that is added by default in v17).

**Comment by IriaAlonso** [comment]
@dalonsod changes done

**Comment by IriaAlonso** [comment]
@dalonsod done

**Review by dalonsod — APPROVED** [review]
:+1:

**Comment by dalonsod** [comment]
@Anxo82 another finished migration

**Pull request merged**